jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/SecurityProviders.java

```diff
@@ -41,13 +41,14 @@ public final class SecurityProviders {
 	private SecurityProviders() {}
 
 	/**
-	 * Resets the security digests during checkpoint.
+	 * Resets all cryptographic state within the CRIUSEC provider that
+	 * was accumulated during checkpoint.
 	 */
 	public static void registerResetDigests() {
```

Comment:
Perhaps this should be more generic too?

Reply:
Perhaps `registerResetState()`?

Reply:
done